tests/backup/backup_resiliency_test.go

```diff
@@ -1025,3 +1026,258 @@ var _ = Describe("{ScaleMongoDBWhileBackupAndRestore}", func() {
+						[]string{namespace}, labelSelectors, orgID, destClusterUid, "", "", "", "", ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Taking backup %s of application- %s", backupName, namespace))
+				}(backupName, namespace)
+			}
+			wg.Wait()
+			log.InfoD("The list of backups taken are: %v", backupNames)
+		})
+		Step("Reboot one worker node on application cluster when backup is in progress", func() {
+			log.InfoD("Switching cluster context to application[destination] cluster")
+			SetDestinationKubeConfig()
+			listOfWorkerNodes = node.GetWorkerNodes()
+			err := Inst().N.RebootNode(listOfWorkerNodes[0], node.RebootNodeOpts{
+				Force: true,
+				ConnectionOpts: node.ConnectionOpts{
+					Timeout:         rebootNodeTimeout,
+					TimeBeforeRetry: rebootNodeTimeBeforeRetry,
+				},
+			})
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Rebooting worker node %v", listOfWorkerNodes[0].Name))
+			log.InfoD("Switching cluster context back to source cluster")
+			err = SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster")
+
+		})
+		Step("Check if backup is successful after one worker node on application cluster is rebooted", func() {
+			log.InfoD("Check if backup is successful after one worker node on application cluster is rebooted")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, backupName := range backupNames {
+				err := backupSuccessCheck(backupName, orgID, maxWaitPeriodForBackupCompletionInMinutes*time.Minute, 30*time.Second, ctx)
+				dash.VerifyFatal(err, nil, "Inspecting backup success for - "+backupName)
+			}
+		})
+		Step("Check if the rebooted node on application cluster is up now", func() {
+			log.InfoD("Check if the rebooted node on application cluster is up now")
+			log.InfoD("Switching cluster context to destination cluster")
+			SetDestinationKubeConfig()
+			listOfWorkerNodes = node.GetWorkerNodes()
+			nodeReadyStatus := func() (interface{}, bool, error) {
+				err := Inst().S.IsNodeReady(listOfWorkerNodes[0])
+				if err != nil {
+					return "", true, err
+				}
+				return "", false, nil
+			}
+			_, err := task.DoRetryWithTimeout(nodeReadyStatus, K8sNodeReadyTimeout*time.Minute, K8sNodeRetryInterval*time.Second)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying the status of rebooted node %s", listOfWorkerNodes[0].Name))
+			err = Inst().V.WaitDriverUpOnNode(listOfWorkerNodes[0], Inst().DriverStartTimeout)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying the node driver status of rebooted node %s", listOfWorkerNodes[0].Name))
+			log.InfoD("Switching cluster context back to source[backup] cluster")
+			err = SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster")
+		})
+		Step("Taking new backup of applications", func() {
+			log.InfoD("Taking new backup of applications")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			dash.VerifyFatal(err, nil, "Getting admin context")
```

Comment:
How about log.FailOnError(err, "Fetching px-central-admin ctx") to maintain consistency? Please refer to PR #1228.

Reply:
done